wai-aria/role/role/role_none_conflict_resolution.tentative.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Name Comp: inherited presentational role with global attr (Tentative)</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+
+    <h1>Presentational role inherited with global</h1>
```

Comment:
Would be good to add a list of links to the portion of the specs this test is validating.